src/main/resources/META-INF/load_mysql.sql

```diff
@@ -0,0 +1,19 @@
+INSERT INTO Plan(NAME, DESCRIPTION) VALUES ('plan1', 'dec1');
```

Comment:
Test data to the `src/test/resources` folder if it is going to be used, otherwise, removed

Reply:
removed